src/graphql/HolofuelActionableTransactionsQuery.gql

```diff
@@ -11,5 +11,7 @@ query HolofuelActionableTransactionsQuery {
     fees
     isPayingARequest
     canceledBy { id, nickname }
+    inProcess
+    actioned
```

Comment:
do we definitley need separate `inProcess` and `actioned` fields? Aren't they negations of each other? Or have a similar relationship that makes having both redundant? Let's talk about this, I might be missing something.